inst/specs/pitching.yml

```
@@ -0,0 +1,100 @@
+df:
+  source: Lahman::Pitching
+  name: arremesadores
+variables:
+  playerID:
+    trans: id_jogador
+    desc: ID do jogador
+  yearID:
+    trans: id_ano
+    desc: "Ano"
+  stint:
+    trans: ordem_equipes
+    desc: "Ordem na qual o jogador se mudou entre equipes na mesma temporada"
+  teamID:
+    trans: id_equipe
+    desc: ID da equipe (fator)
+  lgID:
+    trans: id_liga
+    desc: "ID da liga (fator com os níveis AA, AL, FL, NL, PL, UA)"
+  W:
+    trans: jogos_ganhados
+    desc: Jogos disputados ganhados
+  L:
+    trans: jogos_perdidos
+    desc: Jogos disputados perdidos
+  G:
+    trans: jogos_disputados
+    desc: Jogos disputados
+  GS:
+    trans: jogos_iniciados
+    desc: Jogos disputados iniciados
+  CG:
+    trans: jogos_completos
+    desc: "Número de jogos completos (9 entradas lançadas) que o jogador lançou"
+  SHO:
+    trans: shutouts
+    desc: "Número de shutouts (jogos completos sem permitir corridas) que o jogador lançou"
+  SV:
+    trans: jogos_salvos
+    desc: Jogos salvos
+  IPouts:
+    trans: IPouts
+    desc: "Outs lançadas à equipe adversária (igual a entradas lançadas x 3)"
+  H:
+    trans: hits
+    desc: Hits permitidos pelo oponente
+  ER:
+    trans: corridas_vencidas
+    desc: Corridas limpas recebidas
+  HR:
+    trans: cuadrangulares
+    desc: Cuadrangulares recibidos
+  BB:
+    trans: BB
+    desc: Base nas bolas cedidas ao oponente
+  SO:
+    trans: ponches
+    desc: Ponches dados ao oponente
+  BAOpp:
+    trans: media_rebatida_oponente
+    desc: "Média de rebatidas do oponente"
+  ERA:
+    trans: media_execucoes_limpas
+    desc: "Média de execuções limpas permitidas  (normalizada para 9 entradas lançadas)"
+  IBB:
+    trans: IBB
+    desc: Base em bolas intencionais cedidas ao oponente
+  WP:
+    trans: lances_desviados
+    desc: Lances desviados lançados pelo arremessador
```

Não sei se existe tradução para wild pitch; sugiro manter o termo e explicar brevemente
```suggestion
    desc: Wild pitches (arremessos ruins que permitem o avanço de corredores adversários) lançados pelo arremessador
```